app/models/user_collection.rb

```diff
@@ -20,6 +20,15 @@ def self.add_admin(user_id, collection_id)
     end
   end
 
+  def migrate
+    rolify_fole = if role == "ADMIN"
```

Comment:
Should this be `rolify_role`?

Reply:
Thanks @bess!  That is a typo.  We are not keeping track of Foles 🐴 in the system